tests/Fixtures/app/config/config.yml

```diff
@@ -94,17 +94,17 @@ services:
 
     app.my_dummy_resource.search_filter:
         parent:    'api_platform.doctrine.orm.search_filter'
-        arguments: [ { 'id': 'exact', 'name': 'partial', 'alias': 'start', 'description': 'word_start', 'relatedDummy.name': 'exact', 'relatedDummies': 'exact', 'dummy': 'ipartial' } ]
+        arguments: [ { 'id': 'exact', 'name': 'partial', 'alias': 'start', 'description': 'word_start', 'relatedDummy.name': 'exact', 'relatedDummies': 'exact', 'dummy': 'ipartial', 'embeddedDummy.dummyName': 'partial' } ]
         tags:      [ { name: 'api_platform.filter', id: 'my_dummy.search' } ]
 
     app.my_dummy_resource.order_filter:
         parent:    'api_platform.doctrine.orm.order_filter'
-        arguments: [ { 'id': ~, 'name': 'desc', 'relatedDummy.symfony': ~ } ]
+        arguments: [ { 'id': ~, 'name': 'desc', 'relatedDummy.symfony': ~, 'embeddedDummy.symfony': ~ } ]
```

Comment:
@antograssiot , you're right I test nothing about this. I should order[embeddedDummy.symfony]=asc to respond to your comment above.